composer.json

```diff
@@ -17,7 +17,7 @@
     "require-dev" : {
         "squizlabs/php_codesniffer" : "3.5.3",
         "phpunit/phpunit" : "7.0.0",
-        "facebook/webdriver" : "dev-master",
+        "php-webdriver/webdriver" : "^1.8.2",
```

Comment:
This should not be a hardcoded version number, it should be a version constraint. The `composer.lock` stores the specific version.

Reply:
the `^` character means that it's not hard-coded though. Do you have a suggestion for an alternate version here? I don't have a preference but I'm also not sure what value would be better for you.

Reply:
Maybe `@dev`? Stackoverflow answer says it will get the latest package. I'm unsure if @driusan means that though.

Reply:
I looked up what the '^' means.. the syntax is a little obtuse, but it looks like it's fine.